packages/babel-preset/src/transformPlugin.ts

```diff
@@ -143,6 +151,10 @@ function buildCSSEntriesMetadata(
       return [
         slot,
         cssClasses.map(cssClass => {
+          if (cssClass === null) {
+            return '';
+          }
```

Comment:
```suggestion
```

This shouldn't be necessary, the `classesBySlot` object is created to ignore null values